test/examples/uct_hello_world.c

```diff
@@ -383,6 +401,8 @@ static ucs_status_t dev_tl_lookup(const cmd_args_t *cmd_args,
     return status;
 close_md:
     uct_md_close(iface_p->md);
+release_tl_resources:
+    uct_release_tl_resource_list(tl_resources);
```

Comment:
can we avoid duplication with line 387?

Reply:
re-implemented, could check if it is ok?